src/dashboard/components/files/FileActions.js

```diff
@@ -29,15 +37,46 @@ export default function FileOptions ({missing, id, filePath, openFile}) {
     return <DeleteConfirmModal name={name} onDelete={deleteFile} onCancel={() => setDeleting(false)}/>
   }
 
+  const renameFile = () => {
+    console.log('start function')
+    // first, i need to open the dialog to ask the user where to save the file
+    const fileName = dialog.showSaveDialogSync(win, {filters, title: i18n('Where would you like to move it?')})
+    console.log('dialog')
+    // take the name they chose
+    if (fileName) {
+      let newFilePath = fileName.includes('.pltr') ? fileName : `${fileName}.pltr`
+      // rename it in known_files
+      editKnownFilePath (filePath, newFilePath)
+      // save file to disk:
+      // 1. read contents of the file
+      const contents = fs.readFileSync(filePath, 'utf-8')
+      // 2. save the contents to the new path
+      saveFile(newFilePath, contents)
+      // 3. delete the file at the old path
+      shell.moveItemToTrash(filePath, true)
+      // change the window's title
+      win.setRepresentedFilename(newFilePath)
```

Comment:
oh wait, we don't need this line or the next one either. I thought we did, but that's only for the project windows. The dashboard doesn't do this